src/es/home.html

```diff
@@ -0,0 +1,121 @@
+
+    <dl class="counter">
+      <dt>
+        <a href="/apple-browser-ban">#AppleBrowserBan</a>
+      </dt>
+      <dd><strong>La prohibición de Apple de los navegadores de terceros</strong> en iOS es profundamente anticompetitivo, priva al equipo de Safari/WebKit de financiación y ha estancado la innovación durante los últimos 10 años y ha impedido que las aplicaciones web despeguen en los dispositivos móviles.</dd>
+
+      <dt>Integración profunda de sistemas</dt>
+      <dd><strong>Las aplicaciones web deben convertirse en simples aplicaciones.</strong> Las aplicaciones creadas con la web libre y abierta necesitan un tratamiento y una integración iguales. Los ecosistemas propietarios cerrados y sujetos a impuestos muy altos no deberían recibir ninguna preferencia.</dd>
+
+      <dt>Igualdad en aplicaciones web</dt>
+      <dd><strong>Se deben eliminar todas las barreras artificiales colocadas por los guardianes.</strong> Las aplicaciones web, si se permiten, pueden ofrecer una funcionalidad equivalente con mayor privacidad y seguridad para casos de uso exigentes.</dd>
+    </dl>
+  </div>
+</div>
+
+<div class="get-involved">
+  <div class="who-entry">
+    <div class="group">
+      <h2>Apoyanos</h2>
+      <p>Los reguladores del Reino Unido, la UE, Japón, Australia y los EE. UU. han comenzado a investigar la prohibición de Apple de los motores de búsqueda de terceros y están considerando intervenciones para permitir que las aplicaciones web compitan con las nativas. Lo más importante es que se ha incluido un texto en este sentido en la histórica ley tecnológica de la UE, la Ley de Mercados Digitales.</p>
+
+      <p>Esto no era inevitable. Durante los últimos dos años, Open Web Advocacy ha informado ampliamente a los reguladores y legisladores sobre el potencial de la Web para contrarrestar el duopolio de las aplicaciones móviles.</p>
+
+      <p>Casi con toda seguridad, como resultado directo de nuestra defensa, Apple ha contratado a docenas de empleados adicionales
+para trabajar en WebKit y Safari. Este ancho de banda adicional y la presión regulatoria han contribuido directamente
+a la incorporación de funciones que se habían demorado durante mucho tiempo, como las notificaciones push y las credenciales de aplicaciones para las aplicaciones web.
+Las cifras del personal de WebKit son secretas, pero creemos que se trata de un aumento porcentual significativo.</p>
+
+      <p><strong>La competencia, o al menos la amenaza de ella, funciona.</strong></p>
+
+      <p>OWA necesita tu ayuda para seguir ejerciendo presión para restablecer la competencia y hacer realidad
+las aplicaciones web universales. Puedes marcar una verdadera diferencia para garantizar el futuro de la
+web apoyando esta labor. Dependemos de las donaciones para desafiar a los guardianes increíblemente ricos
+que tienen todos los incentivos para frenar el avance de la web.</p>
+
+      <p>Puedes<a href="/donate">donar</a> o únete a nuestra comunidad en
```

Comment:
```suggestion
      <p>Puedes<a href="/donate">donar</a> o unirte a nuestra comunidad en
```
Keep the same the tone of the verbs